### ISSUE TITLE
word stuck together on man page

### ISSUE THREAD

**Issue by jidanni** [libwww-perl/WWW-Mechanize#65]
```
$ man WWW::Mechanize|perl -nwle 'print for /\S+instances/g;'
WWW::Mechanizeinstances
```
Package: libwww-mechanize-perl
Version: 1.83-1

**Comment by genio** [comment]
Thanks!  Fixed in commit 59bd10e

**Comment by genio** [comment]
Sorry. I meant 5c69a86
